pkg/signals/signal.go

```diff
@@ -21,8 +21,6 @@ import (
 	"os/signal"
 )
 
-var onlyOneSignalHandler = make(chan struct{})
-
```

Comment:
We can remove commented code related to `onlyOneSignalHandler`. Commented code exist at line number [30](https://github.com/openebs/cstor-operators/blob/84ac251371f709e1d12d7f079ea4bb274d81a7a4/pkg/signals/signal.go#L30)